test/swe2d/data/FVCOM.json

```diff
@@ -0,0 +1 @@
+{"h+": [0.85, 0.967, 0.996], "h-": [0.818, 0.963, 0.996], "c+": [0.991, 0.999, 1.0], "c-": [0.991, 0.999, 1.0], "rms": [0.0035165, 0.000609, 0.00010149], "dx": [0.25, 0.125, 0.05], "dt": [0.01, 0.005, 0.002]}
```

Comment:
Maybe in these file names you could somehow indicate that these files are related to the Rossby solition test case?